recipes/keygen-py/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "keygen-py" %}
+{% set version = "0.0.1.dev4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/keygen_py-{{ version }}.tar.gz
+  sha256: a2da9aa961e02e9dd2fe947a1369642feffc83828939bd148b688f3aea33c10c
+
+build:
+  number: 0
+  skip: true  # [py<39 and py>312]
+
+requirements:
+  build:
+    - python
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - maturin
+    - rust
+    - make
+    - perl
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+
+test:
+  imports:
+    - keygen_sh
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/terra-quantum-public/keygen-py
+  summary: Unofficial Keygen SDK for Python. Integrate license activation and offline licensing. Wrapper around keygen-rs rust crate
+  license: MIT
+  license_file: LICENSE
```

Comment:
You need to bundle third-party licenses using `cargo-bundle-licenses`

Reply:
We will add the THIRDPARTY.yml in the next release. Do we need to reference the file somewhere in here?

Reply:
No, please generate it in the recipe.

Reply:
the file should now be included